Point tutorials origin to master

MakisH opened the pull request:
Depends on https://github.com/precice/tutorials/pull/160

shkodm commented:
I've just checked locally. Looks good.

shkodm commented on _codes/dealii.md:
```diff
@@ -0,0 +1,22 @@
+
+[deal.II](https://www.dealii.org/) is an open source software library supporting the creation of finite element codes. It uses state-of-the-art programming techniques to offer a modern interface to the complex data structures and algorithms required.
+
+Build status:<a style="text-decoration: none" href="https://travis-ci.org/precice/dealii-adapter" target="_blank"><img src="https://travis-ci.org/precice/dealii-adapter.svg?branch=master" alt="Build status">
```
Add one space after build status, before travis status. Otherwise the margin is too small.

DavidSCN replied:
Hope it's better now.

shkodm commented:
Also, should some tutorial with `deal.ii` to the `resources` section?

DavidSCN commented:
> Also, should we add some tutorial with deal.ii to the `resources` section?

Good question, maybe @MakisH knows. is this related to #45 ?

DavidSCN commented:
Concerning the version: Most of  the used functions are quiet basic. So, I cannot really say, which deal.II version are going to work and which not. Of course, I could mention there the most recent version the adapter has been built on. But this is probably a restriction.

MakisH commented:
> Concerning the version: Most of the used functions are quiet basic. So, I cannot really say, which deal.II version are going to work and which not. Of course, I could mention there the most recent version the adapter has been built on. But this is probably a restriction.

Still, giving one specific "known-to-work" version to the user is invaluable. I fixed the field and added more information. LGTM.

MakisH committed:
Point tutorials origin to master

MakisH committed:
Tutorials overview: Add how to get the tutorials
